src/vm/gccover.cpp

```diff
@@ -920,8 +921,10 @@ bool replaceInterruptibleRangesWithGcStressInstr (UINT32 startOffset, UINT32 sto
                          *((DWORD*)instrPtr) != INTERRUPT_INSTR_CALL &&
```

Comment:
I think we need the check for even the ARM32 case, right? The isCallToStopForGCJitHelper() for ARM32 may be different. 
We probably are not running GCStress on ARM32 -- because of which it we've not caught the problem.
It would be great to recognize the case here too, if possible.
Otherwise, please add a comment and attach an issue. Thanks.